modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,396 @@
+aws iam put-role-policy --role-name albo-controller --policy-name perms-policy-albo-controller --policy-document file://albo-controller-permission-policy.json
+----
++
+.. Generate the controller's AWS credentials:
++
+[source,terminal]
+----
+$ cat <<EOF > albo-controller-aws-credentials.cfg
+[default]
+sts_regional_endpoints = regional
+role_arn = ${CONTROLLER_ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+----
++
+.. Create the controller's credentials secret by using the generated AWS credentials:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator create secret generic aws-load-balancer-controller-cluster --from-file=credentials=albo-controller-aws-credentials.cfg
+----
++
+. Create an AWS IAM role and assign the ALBO and ALBC AWS IAM policies as defined in the previous steps. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
+. Add the tags necessary for subnet discovery:
+
+.. Add the following `{Key: Value}` tag to the VPC hosting the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster. Replace `{Cluster Infra ID}` with the Infra ID specified previously:
++
+[source, terminal]
+----
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+----
++
+.. Add the following ELBv2 `{Key: Value}` tags to the private subnets and, optionally, to the public subnets:
+
+* Private subnets: `kubernetes.io/role/internal-elb:1`
+* Public subnets: `kubernetes.io/role/elb:1`
++
+[NOTE]
+====
+Internet-facing and internal load balancers will be created within the AZ to which these subnets belong.
+====
++
+For more information on adding tags to AWS resources, including VPCs and subnets, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
++
+[IMPORTANT]
+====
+ELBv2 resources (such as ALBs and NLBs) created by ALBO do not inherit custom tags set for
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+clusters. You must set tags separately for these resources.
+====
++
+. Log in to the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster as a user with the `cluster-admin` role and create a new project using the following command:
```

Comment:
I tested with `dedicated-admin` and it can work, so we can replace `cluster-admin` with it, and please move this to step `2.a`.

Reply:
Just so I am clear, 2a should now be this: 
![image](https://github.com/openshift/openshift-docs/assets/122639474/70fd408c-8d82-4280-a9df-c47a04eeb981)
and step nine should be gone (previous step 10 will now be step 9). Correct? Thanks!

Reply:
Yes, correct.